.changeset/tangy-deer-fetch.md

```diff
@@ -4,3 +4,4 @@
 
 - feat(cli): remove `--no-preconditions` option from `sv add`
 - feat(cli): add `--no-git-check` option to `sv add`. With this flag, even if some files are dirty, no prompt will be shown
+- feat(cli): type `Precondition` was replaced with `Verification`
```

Comment:
Do we need to add a changeset for that? I don't think this type was publicly exposed, even if it couldn't be used since community adders are not exposed.

Reply:
It's exported from core, but yes let's not add it, it's not for users at this point.